### PR TITLE
Pull to refresh on UITableView implemented

### PR DESCRIPTION
**Pull request by diegomedina248**

### CODE REVIEW

File: Project-iOS/XLProjectName/XLProjectName/Helpers/Extensions/UIRefreshControl+Rx.swift [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+//  Created by Diego Medina on 3/29/16.
+//  Copyright © 2016 XLOrganizationName. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+import RxCocoa
+import UIKit
+
+extension UIRefreshControl {
```
Comment (mtnbarreto) [inline Project-iOS/XLProjectName/XLProjectName/Helpers/Extensions/UIRefreshControl+Rx.swift]: why not UIControl?

Comment (diegomedina248) [inline Project-iOS/XLProjectName/XLProjectName/Helpers/Extensions/UIRefreshControl+Rx.swift]: I thought of that, but for UIControl you should have something more generic (i.e. for any event). But, for the UIRefreshControl, i think observing the ValueChanged Event is the most common thing to do.

**Comment by mtnbarreto** [comment]
I didn't take a look at rxCocoa api but rx_controlEvent func seems to apply to any control so your rx_valueChanged helper makes sense for any control.

File: Project-iOS/XLProjectName/XLProjectName/Helpers/Networking/PaginationViewModel.swift [hunk 2]
```diff
@@ -15,9 +15,14 @@ class PaginationViewModel<Element: Decodable where Element.DecodedType == Elemen
     
     var request: PaginationRequest<Element>
     
+    let forceRefreshTrigger = PublishSubject<Void>()
```
Comment (mtnbarreto) [inline Project-iOS/XLProjectName/XLProjectName/Helpers/Networking/PaginationViewModel.swift]: Have you thought about using only one refreshTrigger of type `PublishSubject<bool>`

In this case emitted items should represent if the refresh should cancel pending request or not (force or not).

**Pull request merged**